src/mainMenuScene.js

```diff
@@ -301,20 +304,28 @@ class MainMenuScene {
     }
 
     _onMenuEnter(duration) {
-        let fadeIn = 0;
         const fadeTime = duration || 1500;
         this._menuContainer.isVisible = true;
         this._menuContainer.alpha = 0;
+
+        // To prevent clicking on buttons during transition.
+        this.buttons.forEach((btn) => {
```

Comment:
this could be `this._menuGrid.children.forEach`, or per above comment `this.menuItems.forEach`